config.go

```diff
@@ -126,6 +130,7 @@ func NewConfig() *Config {
 	c.Net.DialTimeout = 30 * time.Second
 	c.Net.ReadTimeout = 30 * time.Second
 	c.Net.WriteTimeout = 30 * time.Second
+	c.Net.KeepAlive = 0 * time.Second
```

Comment:
0 is the default value for uninitialized struct members, so this line is unnecessary